app/models/notification.rb

```diff
@@ -48,10 +48,6 @@ class Notification < ApplicationRecord
 
   cache_associated :from_account, status: STATUS_INCLUDES, mention: [status: STATUS_INCLUDES], favourite: [:account, status: STATUS_INCLUDES], follow: :account
 
-  def activity(eager_loaded = true)
```

Comment:
Right so when you're doing rails preloading using `includes(:model)` but on a polymorphic association, when you're using the polymorphic association method, it loads the record fresh, ignoring the preload. So I overloaded that method to use the preloaded records. However, sometimes they are not preloaded, which is when I'd set the param to false. I don't remember why though.